pkg/venafi/tpp/connector.go

```diff
@@ -414,8 +414,37 @@ func (c *Connector) requestMetadataItems(dn string) ([]metadataKeyValueSet, erro
 	return response.Data, err
 }
 
-// requestSystemVersion returns the TPP system version of the connector context
-func (c *Connector) requestSystemVersion() (string, error) {
```

Comment:
Please don't rename this function, instead you can create the function `RetrieveSefIdentity()` that you need and then call into this the` requestSystemVersion()` function.